apps/condo/domains/ticket/schema/TicketExportTask.js

```diff
@@ -168,6 +169,8 @@ const TicketExportTask = new GQLListSchema('TicketExportTask', {
             },
         },
 
+        parameters: TICKET_EXPORT_PARAMETERS_FIELD,
```

Comment:
`options` field name seems to be simpler to read.

Reply:
it seems like a lot of changes need to be made to change `parameters` to `options` if we want to keep the same names everywhere :)